pkg/resource/events-logger-config/alloy/events-logger.alloy.template

```diff
@@ -37,12 +37,12 @@ loki.process "default" {
 // Loki target configuration
 loki.write "default" {
 	endpoint {
-		url                = nonsensitive(remote.kubernetes.secret.credentials.data["{{ .LoggingURLKey }}"])
+		url                = convert.nonsensitive(remote.kubernetes.secret.credentials.data["{{ .LoggingURLKey }}"])
```

Comment:
I don't get what change this will bring (I'm referring to all added `convert` and `sys` before funcion names)

Reply:
Remove logs in the console about deprecated functions 😂

Reply:
I guess it was caused by deps upgrades, right?
Because when I build from main I can't see the deprecation notices.

Reply:
No those are in alloy logs on WCs. You can check out the logs of a setting alloy pod

Reply:
Oh, it's the alloy config files :facepalm: 

Then yes, it's part of the 1.4.0 changelog I guess: https://github.com/grafana/alloy/blob/main/CHANGELOG.md#other
> Renamed standard library functions. Old names are still valid but are marked deprecated.

Reply:
Yes I had to update it because Nancy was failing the builds